<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="73" started-at="2016-11-15T10:32:16Z" finished-at="2016-11-15T10:32:16Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="73" started-at="2016-11-15T10:32:16Z" finished-at="2016-11-15T10:32:16Z">
      <class name="com.sel.tests.MyTest">
        <test-method status="FAIL" signature="setUp()[pri:0, instance:com.sel.tests.MyTest@51565ec2]" name="setUp" is-config="true" duration-ms="0" started-at="2016-11-15T12:32:06Z" finished-at="2016-11-15T12:32:06Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at com.sel.tests.TestBase.setUp(TestBase.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:277)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="testPageObject()[pri:0, instance:com.sel.tests.MyTest@51565ec2]" name="testPageObject" duration-ms="13" started-at="2016-11-15T12:32:16Z" finished-at="2016-11-15T12:32:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPageObject -->
        <test-method status="SKIP" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.sel.tests.MyTest@51565ec2]" name="tearDown" is-config="true" duration-ms="0" started-at="2016-11-15T12:32:16Z" finished-at="2016-11-15T12:32:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:com.sel.tests.MyTest@51565ec2]" name="tearDown" is-config="true" duration-ms="0" started-at="2016-11-15T12:32:16Z" finished-at="2016-11-15T12:32:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.sel.tests.MyTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
